server/index.js

```diff
@@ -5,26 +5,135 @@ const api = require('./helpers.js');
 
 const app = express();
 
+/* EXPRESS MIDDLEWARE */
+
 app.use(express.static(__dirname + '/../client/dist/'));
 app.use(cors());
 app.use(express.urlencoded({ extended: true }));
 app.use(express.json());
 
 
-/* MAIN PRODUCT ID GET HANDLER */
+/* PRODUCT DATA COMPONENT INITIALIZER */
 
-app.get('/products/:productID', (req, res) => {
-  const productID = req.params.productID;
-  let totalProductInfo = [api.getProductInfo(productID), api.getProductStyles(productID), api.getQuestions(productID), api.getReviews(productID), api.getReviewsMetadata(productID)];
+app.get('/products/:product_id', (req, res) => {
```

Comment:
Is there a particular reason for using snake case here?
If not, I suggest changing it to camel case to keep the same style as the rest of the codebase.